UniMath/CategoryTheory/.package/files

```diff
@@ -18,13 +18,13 @@ limits/initial.v
 limits/graphs/initial.v
 limits/zero.v
 limits/graphs/zero.v
+limits/bincoproducts.v
 limits/coproducts.v
-limits/arbitrary_coproducts.v
-limits/graphs/coproducts.v
+limits/graphs/bincoproducts.v
```

Comment:
I take that back, the standard seems to be to use camelcase, so let's make another PR where we make the CT library filenames camelcase as well.